src/Controller/NetsEasyController.php

```diff
@@ -50,9 +50,12 @@ public static function create(ContainerInterface $container) {
    *   Returns response containing paymentId from Nets endpoint.
    */
   public function createPayment(Request $request) {
-    // @todo Implement error handling.
     $amountToPay = floatval($request->get('amountToPay'));
-    $callbackUrl = $request->get('callbackUrl') ?? NULL;
+    $callbackUrl = $request->get('callbackUrl');
+
+    if (!$callbackUrl || $amountToPay >= 0) {
```

Comment:
This amount check seems wrong. Don't you mean `$amountToPay <= 0`? Remember to test your code.